view/theme/frio/css/style.css

```diff
@@ -1646,6 +1646,11 @@ section #jotOpen {
     z-index: 1000;
 }
 
+/* Space between content and head */
+.wall-spacer {
+	height: 10px;
```

Comment:
Code standards: Wrong indentation, please use the surrounding indentation.

Reply:
Hmm looked fine, when i edited it. Corrected it now. Should be fine now.

Reply:
This still is a tab when indentation is 4 spaces around.

Reply:
I used spaces .. hmm somethings wrong

Reply:
Nope, still a tab.